dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/LogServiceTest.java

```diff
@@ -419,6 +420,16 @@ private void checkForTraceMessage(String msg, int expectedNum) throws Exception
         assertEquals("Did not find the expected number of messages: " + msg, expectedNum, found.size());
     }
 
+    private void checkForTrace(String msg, int expectedNum) throws Exception {
```

Comment:
Should you replace all `checkForTraceMessage()` with `checkForTrace()`?

Reply:
`checkForTraceMessage()` checks string in both message.log and trace.log to see if it exist. I think the function was made generic enough that it would look in both logs. Some tests use it to look for log in message and some only use it to look for log in trace. The failing test only applied to log in trace, so the wait message is only for trace.